### DISCUSSION TITLE
8月16日购买的一年专业版，一直显示免费版，已发帖留言好几天，请尽快帮忙解决，谢谢！账号邮箱85423166@qq.com，支付截图如下

### DISCUSSION THREAD

**Discussion by wander0324** [getlantern/lantern-forum-cn#816] [General]
![付款记录](https://github.com/user-attachments/assets/6932fef3-b5be-4e55-9e42-6dc3c719fa83)

**Comment by MaxMKMax** [comment]
@wander0324 您好. 好的我已经帮您提交问题上去了.

**Reply by wander0324** [reply]
> @wander0324 您好. 好的我已经帮您提交问题上去了.

好的，麻烦尽快帮忙处理一下呢，我留言好几条了，谢谢！
